starter-code/src/components/NewBeer.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from "react";
+import Nav from "./Nav";
+import axios from "axios";
+
+class NewBeer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      name: "",
+      tagline: "",
+      description: "",
+      firstBrewed: 0,
+      brewersTips: "",
+      attenuationLevel: "",
+      contributedBy: ""
+    };
+  }
+
+  handleFormSubmit = () => {
+    e.preventDefault();
+
+    axios.post("", this.state).then(res => {
+      console.log(res.data);
+    });
+
+    this.setState({
+      name: ""
+    });
+  };
+
+  handleChange(e) {
+    let { name, value } = e.target; //this is targeting props from the form below
+
+    this.setState({
+      [name]: value
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <Nav />
+        <form onSubmit={this.handleFormSubmit}>
+          <div className="form-group">
+            <label for="exampleFormControlInput1">Name</label>
+            <input
```

Comment:
you could also create an component that handle Inputs, i saw that you did it in your other project